### PR TITLE
coling-2022-ingestion

### PR DESCRIPTION
**Pull request by xinru1414**

### CODE REVIEW

**Comment by dustalov** [comment]
Hi, will the DOIs be assigned?

**Comment by xinru1414** [comment]
@mjpost Ingestion date is 10-11-2022

File: data/yaml/venues.yaml [hunk 1]
```diff
@@ -216,6 +222,10 @@ codeswitch:
 codi:
   acronym: CODI
   name: Workshop on Computational Approaches to Discourse
+codicrac:
+  acronym: CODICRAC
+  name: The CODI-CRAC 2022 Shared Task on Anaphora, Bridging, and Discourse Deixis
+    in Dialogue
 cogacll:
```
Comment (mjpost) [inline data/yaml/venues.yaml:229]: This is a joint event, not a new venue. It should be handled by:

* Assigning it to the CODI venue ID (e.g., 2022.codi-1)
* Jointly listing it with CRAC (e.g., in joint.yaml, add `2022.codi-1` under `crac`)

Comment (mjpost) [reply data/yaml/venues.yaml:229]: (Note this will require renaming the PDFs and any attachments)

**Comment by mjpost** [comment]
@dustalov We only assign DOIs for ACL events.

File: data/yaml/joint.yaml [hunk 2]
```diff
@@ -240,6 +240,7 @@ cogalex:
 - W14-47
 - W16-53
 coling:
+  2022: [2022.smm4h-1, 2022.creativesumm-1, 2022.ccgpk-1, 2022.codi-1, 2022.codicrac-1, 2022.cai-1, 2022.crac-1, 2022.latechclfl-1, 2022.loresmt-1, 2022.fieldmatters-1, 2022.pandl-1, 2022.mmmpie-1, 2022.sdp-1, 2022.textgraphs-1, 2022.trac-1, 2022.tu-1, 2022.vardial-1, 2022.wnut-1, 2022.wat-1]
   2020: [2020.nlp4call-1, 2020.argmining-1, 2020.cogalex-1, 2020.crac-1, 2020.dmr-1, 2020.ecomnlp-1, 2020.finnlp-1, 2020.fnp-1, 2020.gebnlp-1, 2020.lantern-1, 2020.latechclfl-1, 2020.law-1, 2020.msr-1, 2020.mwe-1, 2020.nlp4if-1, 2020.peoples-1, 2020.rdsm-1, 2020.semeval-1, 2020.smm4h-1, 2020.starsem-1, 2020.textgraphs-1, 2020.udw-1, 2020.vardial-1, 2020.wanlp-1]
```
Comment (mjpost) [inline data/yaml/joint.yaml:244]: Missing `2022.argmining-1`

**Comment by dustalov** [comment]
> @dustalov We only assign DOIs for ACL events.

Is this a new policy? For example, COLING '20 has DOIs, e.g., https://aclanthology.org/2020.coling-main.1 (as well as many other papers in https://github.com/acl-org/acl-anthology/blob/master/data/xml/2020.coling.xml).

File: data/yaml/venues.yaml (hunk 1 above)
Comment (xinru1414) [reply data/yaml/venues.yaml]: @mjpost I also added `2022.codi-crac` under `crac` in joint.yaml. Correct me if I'm wrong.

**Comment by nschneid** [comment]
> > @dustalov We only assign DOIs for ACL events.
> 
> Is this a new policy? For example, COLING '20 has DOIs, e.g., https://aclanthology.org/2020.coling-main.1 (as well as many other papers in https://github.com/acl-org/acl-anthology/blob/master/data/xml/2020.coling.xml).

According to https://aclanthology.org/faq/,

> Note while the ACL Anthology hosts third-party materials coming from sister societies, these materials are hosted courtesy the ACL but are not assigned DOIs by the ACL due to costs and copyright limitations

I think the expectation is that it is up to COLING to assign its own DOIs.

**Comment by xinru1414** [comment]
@mjpost ready for review + merge

**Comment by mjpost** [comment]
@dustalov @nschneid We may have done this in 2020 due to oversight or perhaps via special favor. We could arrange this again but DOIs cost $1 each so we would need to arrange for COLING to reimburse ACL, which is best handled over email.

**Review by mjpost — APPROVED** [review]

File: data/yaml/joint.yaml [hunk 3]
```diff
@@ -240,7 +240,7 @@ cogalex:
 - W14-47
 - W16-53
 coling:
-  2022: [2022.smm4h-1, 2022.creativesumm-1, 2022.ccgpk-1, 2022.codi-1, 2022.cai-1, 2022.crac-1, 2022.latechclfl-1, 2022.loresmt-1, 2022.fieldmatters-1, 2022.pandl-1, 2022.mmmpie-1, 2022.sdp-1, 2022.textgraphs-1, 2022.trac-1, 2022.tu-1, 2022.vardial-1, 2022.wnut-1, 2022.wat-1. 2022.argmining-1]
+  2022: [2022.smm4h-1, 2022.creativesumm-1, 2022.ccgpk-1, 2022.codi-1, 2022.codi-crac, 2022.cai-1, 2022.crac-1, 2022.latechclfl-1, 2022.loresmt-1, 2022.fieldmatters-1, 2022.pandl-1, 2022.mmmpie-1, 2022.sdp-1, 2022.textgraphs-1, 2022.trac-1, 2022.tu-1, 2022.vardial-1, 2022.wnut-1, 2022.wat-1. 2022.argmining-1]
```
Comment (xinru1414) [inline data/yaml/joint.yaml:243]: @mjpost added codi-crac to coling, relevant to #2187

File: data/yaml/joint.yaml [hunk 4]
```diff
@@ -324,6 +324,9 @@ corbon:
 cosli:
 - W13-07
 crac:
+  2022:
```
Comment (xinru1414) [inline data/yaml/joint.yaml:327]: @mjpost added codi-crac and codi to crac, relevant to https://github.com/acl-org/acl-anthology/pull/2187

**Pull request merged**